src/features/WorldMap/lineDraw.jsx

```diff
@@ -17,13 +17,41 @@ const borderLineWidth = 0.5;
 const zoomLineStrength = 0.5;
 */
 
+export const colorScheme = {
+  left: '#ef4400',
+  middle: '#f7f7f7',
+  right: '#0083cf',
+  selectedCountry: '#00A600',
+  hoveredCountry: '#00CC00',
+  //"#D0D0D0"
+  noData: 'gray',
+};
+
+function valToColor(value, range) {
+  if (value == null)
+    return colorScheme.noData;
+  var relative_value;
+  if(range.selected !== null) {
+    relative_value = (value - range.selected) / (range.max - range.min);
+  } else {
+    // If range.selected is null, we have our reference at 0.
+    // But, this means that the extreme ends are only "half the bar" away from the reference.
+    // Therefore, we multiply by 2
+    relative_value = 2 * value / (range.max - range.min);
```

Comment:
What do you think of this? In essence, this makes the extreme colors significantly more pronounced and increases the contrast.
I think we should not do this once a country has been selected as in this case, the selected "center point" is not necessarily in the center leading to saturation at one end or a nonlinear scale (compressed at one side). I think the lower contrast also emphasizes that the countries are very similar in most cases.